app_dart/lib/src/model/appengine/branch.dart

```diff
@@ -42,3 +46,27 @@ class Branch extends Model<String> {
     return buf.toString();
   }
 }
+
+/// The serialized representation of a [Branch].
+// TODO(tvolkert): Directly serialize [Branch] once frontends migrate to new serialization format.
+@JsonSerializable(createFactory: false, ignoreUnannotated: true)
+class SerializableBranch {
+  const SerializableBranch(this.branch);
+
+  final Branch branch;
+
+  @JsonKey(name: 'Key')
```

Comment:
Avoid adding `JsonKey`. The reason we have these littered about is we used to run a Go codebase at the same time where the default generated it that way